Automattic/wp-calypso
# Calypso & wp-admin isomorphic extensions

Continues the work from https://github.com/Automattic/wp-calypso/pull/11620

The intent of this PR is to build an example of an extension that can be used both in calypso and wp-admin, so we could use a single codebase to build both versions of a plugin. 

It requires a small php plugin on wp-admin side to work: https://gist.github.com/johnHackworth/73cc8f8bf11e8459544e2361e36eb9e5

How to test:
---------------
0. You will need to edit JETPACK_DIR in `bin/copy-files.sh` to add the path where your js bundles are going to be exported (probably you want them to be in the local mirror of the path where the PHP plugin sits)
0.5 Copy the php plugin in your test site plugins folder and activate it
1. Run `npm run build-jetpack && ./bin/copy-files.sh`
2. Sync your files to your remote site
3. Run calypso locally
4. In your remote site, you should have a new wp-admin menu item "jetpack-ui" you can click to get to the extension
5. In your local calypso, you can go to http://calypso.localhost:3000/test to see the same extension working on calypso
